### ISSUE TITLE
Improve seed puzzle generation.

### ISSUE THREAD

**Issue by vaithak** [vaithak/Sudoku-Generator#4]
The function used for generating the seed is here:  https://github.com/vaithak/Sudoku-Generator/blob/master/sudokuGen.cpp#L136.

**Comment by cchiu727** [comment]
Hi, is this issue still open? I'd like to contribute

**Comment by vaithak** [comment]
Hi @cchiu727, yes this issue is open. It would be great if we can first discuss some potential approaches and then go ahead with implementing it.

**Comment by cchiu727** [comment]
Yes, I'd be glad to. I will take a closer look at the code and reply with some ideas very soon. 👍

**Comment by vaithak** [comment]
Any updates @cchiu727 ?

**Comment by cchiu727** [comment]
Hi @vaithak , I'm sorry for the absence, I am not able to work on this issue anymore
